packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -201,19 +316,72 @@ class _CupertinoCheckboxState extends State<CupertinoCheckbox> with TickerProvid
   @override
   bool? get value => widget.value;
 
-  void onFocusChange(bool value) {
-    if (focused != value) {
-      focused = value;
+  WidgetStateProperty<Color> get _widgetFillColor {
+    return WidgetStateProperty.resolveWith((Set<WidgetState> states) {
+      if (states.contains(WidgetState.disabled)) {
+        return widget.inactiveColor ?? CupertinoColors.white;
+      }
+      if (states.contains(WidgetState.selected)) {
+        return widget.activeColor ?? CupertinoColors.activeBlue;
+      }
+      return CupertinoColors.white;
```

Comment:
> Should this be changed to Return widget.inactiveColor ?? CupertinoColors.white

inactiveColor is the color shown when disabled, not when unchecked. 

> Also, should it default to white? Before it was inactiveGrey which I think was picked to look alright across both dark and light mode. If not, the documentation for inactiveColor says it will default to showing inactiveGrey which it doesn't look like it would be possible.

Previously the transparent color was the default, but I added a new color similar to `CupertinoColors.inactiveGrey` that resolves closer to native in light vs dark modes.

Reply:
> inactiveColor is the color shown when disabled, not when unchecked.

Theoretically it should be when it's unchecked: https://api.flutter.dev/flutter/material/ToggleablePainter/inactiveColor.html

Reply:
Which is not the best naming. It's used closer to "unselectedColor" not as a "inactiveColor".

Reply:
I'd strongly support changing `inactiveColor` to mean the unselected color and being consistent, and either:
a) migrating users to a new parameter `disabledColor`, or 
b) not create a new parameter and allow users configure the color in the disabled state using `fillColor`.

Reply:
I don't think we should use a new parameter, we should have the fillColor do everything. Looks like it was using inactiveGrey before for the border, and just having no color/transparency for the fill.

Reply:
> inactiveColor is the color shown when disabled, not when unchecked.

Ah, I was wrong here. It is the color of the border when unchecked, as you mentioned @MitchellGoodwin. I think I was looking too closely at my changes instead of the current behavior.

Reply:
> I don't think we should use a new parameter, we should have the fillColor do everything. Looks like it was using inactiveGrey before for the border, and just having no color/transparency for the fill.

Another argument for deprecating `inactiveColor`: Material's `Checkbox` doesn't have an `inactiveColor` property, so this would be an opportunity to bring their APIs closer together :)

Reply:
💯  Exactly!